.changeset/light-pears-hammer.md

```diff
@@ -5,6 +5,7 @@
 "section": deprecation
 ---
 
-`supportedFeatures` is deprecated in `IContainerContext`
+The IContainerContext.supportedFeatures property is now deprecated
```

Comment:
Nit: still seems like we should retain the `code` formatting for a reference to an API member, IMO
```suggestion
The `IContainerContext.supportedFeatures` property is now deprecated
```